infrastructure/httpapi/handlers/validators.go

```diff
@@ -298,6 +311,47 @@ func (handler *Validators) getAverageBlockTime() (*big.Float, error) {
 		big.NewFloat(1000),
 	)
 
+	// totalBlockCount > nRecentBlocks: then re-calculate averageBlockTime
+	// totalBlockCount <= nRecentBlocks: use the above calculated averageBlockTime
+	nRecentBlocks := big.NewInt(nRecentBlocksInInt)
+	if totalBlockCount.Cmp(nRecentBlocks) == 1 {
```

Comment:
Can we use a bool var to name this condition and use in here? like isTotalBlockCountExcess = (totalBlockCount.Cmp(nRecentBlocks) == 1)

Reply:
Sure! Will change it!